components/datepicker/src/auro-calendar.js

```diff
@@ -255,12 +260,17 @@ export class AuroCalendar extends RangeDatepicker {
         composed: true,
       }));
     });
+  }
 
-    this.utilCalRender.determineNumCalendarsToRender(this);
+  toggleVisibility(visibility) {
```

Comment:
Does visibility need to be passed in for a toggle? Since we are talking about a boolean why not just have the function set the value to the opposite of the current value?

Reply:
you are right! this function is removed and now just simply setting visible variable